### PR TITLE
ospatching Linux VM

### PR DESCRIPTION
**Pull request by thomas1206**
migrate the ospatching template to the new repo.
Change the parameter according to the following breaking change:
- https://github.com/Azure/azure-quickstart-templates/blob/master/breaking-change.md

### CODE REVIEW

File: vm-ospatching/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+      "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
```
Comment (singhkay) [inline vm-ospatching/azuredeploy.json]: No default locations

File: vm-ospatching/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+            }
+        }
+    },
+    "variables": {
+        "imagePublisher": "Canonical",
+        "imageOffer": "UbuntuServer",
+        "OSDiskName": "osdiskforospatching",
+        "nicName": "myVMNic",
+        "extensionName": "OSPatching",
+        "addressPrefix": "10.0.0.0/16",
```
Comment (singhkay) [inline vm-ospatching/azuredeploy.json]: rename this to vnetAddressPrefix for clarity

File: vm-ospatching/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+        "subnetName": "Subnet",
+        "subnetPrefix": "10.0.0.0/24",
+        "storageAccountType": "Standard_LRS",
+        "publicIPAddressName": "myPublicIP",
+        "publicIPAddressType": "Dynamic",
+        "vmStorageAccountContainerName": "vhds",
+        "vmName": "MyOSPatchingVM",
+        "vmSize": "Standard_D1",
```
Comment (singhkay) [inline vm-ospatching/azuredeploy.json]: Make this a parameter with a default. Then the user has a chance to change the size before deployment

File: vm-ospatching/metadata.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Create a Ubuntu VM with OSPatching extension",
```
Comment (singhkay) [inline vm-ospatching/metadata.json]: Suggestion for name: OS Patching extension on a Ubuntu VM

File: vm-ospatching/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Simple deployment of an Ubuntu VM with OS Patching extension
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fthomas1206%2Fazure-quickstart-templates%2Fmaster%2Fvm-ospatching%2Fazuredeploy.json" target="_blank">
```
Comment (johngossman) [inline vm-ospatching/README.md]: Need to update this url to point to the Azure repo and of thomas1206

File: vm-ospatching/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+      "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+      "contentVersion": "1.0.0.0",
+      "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed."
+            }
+        },
+        "location": {
```
Comment (johngossman) [inline vm-ospatching/azuredeploy.json]: Should provide a set of allowedValues

File: vm-ospatching/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+                    "description": "Password for the Virtual Machine."
+                }
+        },
+        "dnsNameForPublicIP": {
+            "type": "string",
+                "metadata": {
+                      "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+                }
+        },
+        "vmSize": {
```
Comment (johngossman) [inline vm-ospatching/azuredeploy.json]: Should provide allowedValues

File: vm-ospatching/azuredeploy.json (hunk 6 above)
Comment (thomas1206) [inline vm-ospatching/azuredeploy.json]: OK. Added.

**Pull request merged**